imageio/__main__.py

```diff
@@ -0,0 +1,137 @@
+    """
+    if plugin_names.count("all"):
+        # Use all plugins
+        plugin_names = PLUGINS_WITH_BINARIES
+    
+    plugin_names.sort()
+    print("Ascertaining binaries for: {}.".format(", ".join(plugin_names)))
+    
+    if package_dir:
+        # Download the binaries to the `resources` directory
+        # of imageio. If imageio comes as an .egg, then a cache
+        # directory will be created by pkg_resources (requires setuptools).
+        # see `imageio.core.util.resource_dirs`
+        # and `imageio.core.utilresource_package_dir`
+        directory = util.resource_package_dir()
+    else:
+        directory = None
+    
+    for plg in plugin_names:
+        msg = "Plugin {} is not registered for binary download!".format(plg)
+        assert plg in PLUGINS_WITH_BINARIES, msg
+        mod = getattr(plugins, plg)
+        mod.download(directory=directory)
+
+
+def download_bin_main():
+    """ Argument-parsing wrapper for `download_bin` """
+    description = "Download plugin binary dependencies"
+    phelp = "Plugin name for which to download the binary. "\
+            + "If no argument is given, all binaries are downloaded."
+    dhelp = "Download the binaries to the package directory "\
+            + "(default is the users application data directory). "\
+            + "This might require administrative rights."
+    example_text = "examples:\n"\
+                   + "  imageio_download_bin all\n"\
+                   + "  imageio_download_bin ffmpeg\n"\
+                   + "  imageio_download_bin avbin ffmpeg\n"
+    parser = argparse.ArgumentParser(
+        description=description,
+        epilog=example_text,
+        formatter_class=argparse.RawDescriptionHelpFormatter)
+    parser.add_argument("plugin", type=str, nargs="*", default="all",
+                        help=phelp)
+    parser.add_argument("--package-dir", dest="package_dir",
+                        action="store_true", default=False, help=dhelp)
+    args = parser.parse_args()
+    download_bin(plugin_names=args.plugin, package_dir=args.package_dir)
+
+
+def remove_bin(plugin_names=["all"]):
+    """ Remove binary dependencies of plugins
+    
+    This is a convenience method that removes all binaries
+    dependencies for plugins downloaded by imageio.
+    
+    Notes
+    -----
+    It only makes sense to use this method if the binaries
+    are corrupt.
+    """
+    if plugin_names.count("all"):
+        # Use all plugins
+        plugin_names = PLUGINS_WITH_BINARIES
+
+    print("Removing binaries for: {}.".format(", ".join(plugin_names)))
+
+    rdirs = util.resource_dirs()
+
+    for plg in plugin_names:
+        msg = "Plugin {} is not registered for binary download!".format(plg)
+        assert plg in PLUGINS_WITH_BINARIES, msg
+    
+    for rd in rdirs:
+        # plugin name is in subdirectories
+        for rsub in os.listdir(rd):
+            if rsub in plugin_names:
+                shutil.rmtree(op.join(rd, rsub))
```

Comment:
Maybe have a try-except around this? When the lib is still loaded, a remove may fail, preventing the removal for other dirs. Perhaps raise error at the end when any of the deletions failed so that the process returns -1.